microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -125,7 +125,7 @@ class Xfstesting(TestSuite):
     # commit d0c7feaf8767 ("xfs: add agf freeblocks verify in xfs_agf_verify")
     # TODO: will figure out the detailed reason of every excluded case.
     excluded_tests = (
-        "generic/211 generic/430 generic/431 generic/434 xfs/438 xfs/490"
+        "generic/211 generic/430 generic/431 generic/434 generic/738 xfs/438 xfs/490"
```

Comment:
Add comments above, why 738 is removed. Is the timeout resoled after this test is removed?